File: crates/tau-ts-extract/tests/fixtures/models_conformance/project.ts
export const projectModels = models({
    haiku: { backend: "anthropic", model: "claude-haiku-4-5" },
    opus: { backend: "anthropic", model: "claude-opus-4-8" },
});

export const fast = agent({
    display_name: "Fast",
    package: "research@^0.1",
    model: "haiku",
});

export const deep = agent({
    display_name: "Deep",
    package: "research@^0.1",
    model: "opus",
});

File: crates/tau-sdk-codegen/tests/fixtures/basic_agent/project.py
from tau_sdk import agent, models, model, print_toml

m = models(haiku=model(backend="anthropic", model="claude-haiku-4-5"))
fast = agent(display_name="Fast", package="research@^0.1", model="haiku")

print_toml(project="basic-agent", models=m, agents={"fast": fast})
